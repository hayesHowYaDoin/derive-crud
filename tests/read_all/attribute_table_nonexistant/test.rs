use derive_crud::ReadAll;

#[derive(ReadAll)]
#[crud_table("nonexistant_table")]
struct Test {
    #[crud_id]
    id: i64,
    name: String,
    location: String,
}

#[tokio::main]
async fn main() {
    let _results = Test::read_all().await;
}
