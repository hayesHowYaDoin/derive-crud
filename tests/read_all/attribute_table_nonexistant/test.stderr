error: error returned from database: (code: 1) no such table: nonexistant_table
 --> tests/read_all/attribute_table_nonexistant/test.rs:3:10
  |
3 | #[derive(ReadAll)]
  |          ^^^^^^^
  |
  = note: this error originates in the macro `$crate::sqlx_macros::expand_query` which comes from the expansion of the derive macro `ReadAll` (in Nightly builds, run with -Z macro-backtrace for more info)
